zio-http/src/main/scala/zhttp/http/Status.scala

```diff
@@ -3,6 +3,10 @@ package zhttp.http
 import io.netty.handler.codec.http.HttpResponseStatus
 
 sealed trait Status extends Product with Serializable { self =>
+  def toApp: UHttpApp = HttpApp.status(self)
+
+  def toResponse: UResponse = Response(self)
+
```

Comment:
a bit of scala docs would be great!

Reply:
Sure, added some brief doc in the last commit